client/ayon_core/plugins/publish/extract_usd_layer_contributions.py

```diff
@@ -458,7 +458,22 @@ def get_or_create_instance(self,
         return new_instance
 
     @classmethod
-    def get_attribute_defs(cls):
+    def get_attr_defs_for_instance(cls, create_context, instance):
+        # Filtering of instance, if needed, can be customized
+        if not cls.instance_matches_plugin_families(instance):
+            return []
+
+        # Attributes logic
+        disabled = False
```

Comment:
This variable should be `hidden`, or better `visible`.

Reply:
Fixed with https://github.com/ynput/ayon-core/pull/960/commits/d797b78fcaa1bdbc3836036608744318a36f1045